ecosystem/typescript/sdk/src/tests/e2e/coin_client.test.ts

```diff
@@ -19,10 +20,10 @@ test(
     await faucetClient.fundAccount(alice.address(), 100_000_000);
     await faucetClient.fundAccount(bob.address(), 0);
 
-    const txnHash1 = await coinClient.transfer(alice, bob, 42);
+    const txnHash1 = await coinClient.transfer(alice, bob, 42, { coinType: APTOS_COIN });
```

Comment:
Can we update the your coin example to use the transfer function from coin_client?

Reply:
Ok, I was going to put it in here, but the test case requires publishing and a few other logistical things, so I'll just file an issue for it so that we can better track all the test coverage we're missing